chatsky/conditions/slots.py

```diff
@@ -36,3 +36,31 @@ async def call(self, ctx: Context) -> bool:
             return all(manager.is_slot_extracted(slot) for slot in self.slots)
         elif self.mode == "any":
             return any(manager.is_slot_extracted(slot) for slot in self.slots)
+
+
+class GroupSlotsExtracted(BaseCondition):
+    """
+    Check if :py:attr:`.slots` are extracted.
+
+    :param mode: Whether to check if all slots are extracted or any slot is extracted.
+    """
+
+    slots: List[SlotName]
+    """
+    Names of the slots that need to be checked.
+    """
+    required: List[SlotName]
+    """
+    If required slots are extracted condition will return true despite non-required slots being extracted or not.
+    By default set to `["all"]` that sets all slots to required.
+    """
+
+    def __init__(self, *slots: SlotName, required: List[str] = ["all"]):
+        super().__init__(slots=slots, required=required)
+
+    async def call(self, ctx: Context) -> bool:
+        manager = ctx.framework_data.slot_manager
+        if self.required == ["all"]:
+            return all(manager.is_slot_extracted(slot) for slot in self.slots)
+        else:
+            return all(manager.is_slot_extracted(slot) for slot in self.required)
```

Comment:
Wait, what is the point of this condition?
Isn't `GroupSlotsExtracted("slot", required=["slot.subslot"])` equivalent to `SlotsExtracted("slot.subslot")`?

Reply:
Yes indeed. It is supposed to be used like 
`GroupSlotsExtracted(["group_slot_1", "group_slot_2", "group_slot_3"], required=["group_slot_1.subslot_2", "group_slot_2.subslot_3"])`

Possibly can be merged with `SlotsExtracted` in this case?

Reply:
That's still easier to accomplish with `SlotsExtracted`:
```
SlotsExtracted(["group_slot_1.subslot_2", "group_slot_2.subslot_3", "group_slot_3"])
```